MediaBrowser.Providers/Music/MusicBrainzAlbumProvider.cs

```diff
@@ -59,7 +59,7 @@ public class MusicBrainzAlbumProvider : IRemoteMetadataProvider<MusicAlbum, Albu
             _logger = logger;
             _json = json;
 
-            MusicBrainzBaseUrl = configuration["MusicBrainz:BaseUrl"];
+            MusicBrainzBaseUrl = configuration["MusicBrainz_BaseUrl"];
```

Comment:
why replace `:` with `_`?

Reply:
Configuration vie env var didn't like the `:`